index.html

```diff
@@ -86,16 +86,19 @@
     <section>
       <h2>Introduction</h2>
       <p>
-        There is a need in the media industry for an API to support metadata
-        events synchronized to audio or video media, specifically for both
-        <a>out-of-band</a> event streams and <a>in-band</a> discrete events
-        (for example, MPD and <code>emsg</code> events in MPEG-DASH).
-        These <a>media timed events</a> can be used to support use cases
-        such as dynamic content replacement, ad insertion, or presentation of
-        supplemental content alongside the audio or video, or more generally,
-        making changes to a web page, or executing application code triggered
-        from JavaScript events, at specific points on the <a>media timeline</a>
-        of an audio or video media stream.
+        There is a need in the media industry for an API to support timed
+        metadata, i.e., metadata that is synchronized to audio or video media,
+        and used to support use cases such as dynamic content replacement,
+        ad insertion, presentation of supplemental content alongside the audio
+        or video, or more generally, making changes to a web page, or executing
+        application code triggered from DOM events at specific points on the
+        <a>media timeline</a> of an audio or video media stream.
+      </p>
+      <p>
+        Timed metadata may be carried either "in-band", meaning that they are
+        delivered within the audio or video media container or multiplexed with
+        the media stream, or "out-of-band", meaning that they are delivered
+        externally to the media container or media stream.
```

Comment:
I suggest to drop quotes around "in-band" and "out-of-band" and to wrap them in `<a>` instead to link back to the definition of these terms in this document.

Reply:
Done